rush/rush/src/utilities/InstallManager.ts

```diff
@@ -56,25 +68,22 @@ export default class InstallManager {
   /**
    * Returns a map of all direct dependencies that only have a single semantic version specifier
```

Comment:
>direct dependencies that only have a single semantic version specifier [](start = 26, length = 70)

Is that the definition of "implicitly pinned version"?

What is the caller expected to do with the map?

What would happen if we didn't do anything with implicitly pinned dependencies?

This little hack is growing into a feature, so it would be helpful to have a little blurb somewhere that explains the problem and defines the terminology.  When some threshold is reached, it probably needs to come out of InstallManager into its own class.